tsconfig.json

```diff
@@ -18,7 +18,7 @@
     "esModuleInterop": true
   },
   "include": [
-    "src/**/*"
```

Comment:
Presumably we want to include `src/**/*` too? Also, I don't think we want to include tests in the compiled source.

Reply:
There's no `src` directory in this repo. And we don't have a `main` key defined in `package.json`, so there's no compilation target that someone could consume.

Happy to create a `tsconfig.eslint.json` file for this directory, but it seemed like overkill.

Reply:
Ahh sorry, I had an old version of this pulled down locally, which had `src/` still.